.env.test

```diff
@@ -4,8 +4,13 @@ FLASK_ENV=development
 FLASK_SKIP_DOTENV=True
 
 # Change the following values for local development.
+LOGIN_DISABLED=False
 SECRET_KEY=secret-key
 API_KEY=api123
 TOKEN=token123
 BOARD_ID=board123
-TODO_LIST_ID=456
\ No newline at end of file
+TODO_LIST_ID=456
+MONGO_DB_CONNECTION=Test_Database_Connection
+MONGO_DB_DATABASE=Test_Database
```

Comment:
You don't need API_KEY, TOKEN, BOARD_ID, TODO_LIST_ID, MONGO_DB_CONNECTION or MONGO_DB_DATABASE anymore, right?

And `.env.template` is out of date